roles/restic_backup/tasks/main.yaml

```diff
@@ -10,29 +10,29 @@
 
 - name: Set up backup script
   ansible.builtin.copy:
-    src: files/backup.sh
-    dest: /usr/local/sbin/backup.sh
+    src: files/restic_backup.sh
+    dest: /usr/local/sbin/restic_backup.sh
     owner: root
     group: root
     mode: '0775'
 
 - name: Save restic password
   ansible.builtin.copy:
-    content: '{{ backup__restic_password }}'
-    dest: /usr/local/etc/restic_password
+    content: '{{ restic_backup__password }}'
+    dest: /etc/restic_backup.password
     owner: root
     group: root
     mode: '0600'
   no_log: true
-  when: backup__restic_password
+  when: restic_backup__password
 
 - name: Set up cron job
   ansible.builtin.copy:
     content: |
       #!/bin/sh
 
-      export RESTIC_PASSWORD_FILE=/usr/local/etc/restic_password
-      export RESTIC_REPOSITORY={{ backup__restic_repository }}
+      export RESTIC_PASSWORD_FILE=/etc/restic_backup.password
+      export RESTIC_REPOSITORY={{ restic_backup__repository }}
       exec chronic /usr/local/sbin/backup.sh
     dest: /etc/cron.daily/backup
```

Comment:
I think this line should be also changed (in line 14 it is copied with `restic_backup.sh` name):
```
exec chronic /usr/local/sbin/restic_backup.sh
```
Not sure if the next line should be changed as well to rename the cron as well:
```
dest: /etc/cron.daily/restic_backup
```

Reply:
Good catch, thanks!